examples/Demo/FluentUI.Demo.Client/Documentation/Components/Slider/FluentSlider.md

```diff
@@ -52,17 +74,10 @@ If you add content that is **not part** of `Slot="thumb"`, it will be ignored.
 
 {{ SliderThumbs }}
 
-## ReadOnly
-
-{{ SliderReadOnly }}
-
-## EventCallback
-
-{{ SliderEventCallbacks }}
-
 ## Know restrictions
 
-At this point, the `FluentSliderLabel` elements cannot be applied.
+At this moment, the `FluentSliderLabel` elements, included in the v4 library, are not yet supported.
+The `FluentSliderLabel` elements are used to display the value of the slider steps, below the slider.
```

Comment:
The `FluentSliderLabel` component (used to display the value of the slider steps) from version 4 is no longer supported.